### PR TITLE
running local envoy-control with docker

### PR DESCRIPTION
**Pull request by wookieJ**

### CODE REVIEW

**Comment by slonka** [comment]
How does this tie to publishing images to dockerhub? Can dockerhub be configured to use `tools/envoy-control/Dockerfile` path?

**Comment by wookieJ** [comment]
Exactly, we can configure path to `tools/envoy-control/Dockerfile`

**Comment by slonka** [comment]
Can you send a link to an image that is on dockerhub? Can we create images from PRs?

**Comment by wookieJ** [comment]
There is no image yet, because automated build is set for master branch now (run after merge or triggered manually). It is possible to build image easily for every branch, but not for PRs only. We would create own hook for this: https://docs.docker.com/docker-hub/builds/advanced/#custom-build-phase-hooks.

File: tools/envoy-control/Dockerfile [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
 
-RUN mv /var/tmp/envoy-control-runner* /var/tmp/envoy-control-runner
-RUN mkdir /var/tmp/config
+FROM adoptopenjdk/openjdk11:alpine-jre
 
-VOLUME /var/tmp/config
+RUN mkdir /tmp/envoy-control-dist /tmp/envoy-control /bin/envoy-control /etc/envoy-control /var/tmp/config
+COPY --from=builder /home/gradle/src/envoy-control-runner/build/distributions/ /tmp/envoy-control-dist
+COPY ./envoy-control-runner/src/main/resources/application-docker.yaml /etc/envoy-control/application.yaml
+RUN tar -xf /tmp/envoy-control-dist/envoy-control-runner-0.1.0*.tar -C /tmp/envoy-control
```
Comment (lukidzi) [inline tools/envoy-control/Dockerfile:16]: if we merge these 2 `RUN` image will be  lighter:
```
RUN tar -xf /tmp/envoy-control-dist/envoy-control-runner-0.1.0*.tar -C /tmp/envoy-control \
    && mv /tmp/envoy-control/envoy-control-runner*/ /bin/envoy-control/envoy-control-runner
```

**Pull request merged**